experimental/sql/tools/rel_opt.py

```diff
@@ -32,6 +33,7 @@ def register_all_passes(self):
     self.available_passes['alg-to-ssa'] = alg_to_ssa
     self.available_passes['ssa-to-impl'] = ssa_to_impl
     self.available_passes['impl-to-iterators'] = impl_to_iterators
+    self.available_passes['optimize-projections'] = optimize_projections
```

Comment:
I *think* this is effectively the textbook projection push-down. Shouldn't we call it that way?